speechbrain/utils/fetching.py

```diff
@@ -58,38 +158,68 @@ def fetch(
     use_auth_token=False,
     revision=None,
     huggingface_cache_dir=None,
+    local_strategy: LocalStrategy = LocalStrategy.SYMLINK,
```

Comment:
I'm curious if we still want the default to be `SYMLINK` here. I appreciate that we might want to keep the default the same but perhaps we could add a warning and change it in the next version?

Reply:
I thought about this more and I think it might make sense to switch to `NO_LINK` by default. I was wary of doing such a change but I don't think most things implicitly rely on this, and it's usually a simple fix or workaround.

CI passes when changing the default after one change in parameter transfer logic.

It should be plenty documented though... I don't want users to be too surprised when they see an empty directory where the pretrained stuff should be.